.travis.yml

```diff
@@ -19,7 +18,7 @@ matrix:
     script:
     - cargo clippy -- -D clippy-pedantic
   - env: TARGET=x86_64-unknown-linux-gnu
-    rust: 1.25.0
+    rust: nightly
```

Comment:
The problem with switching to Rust nightly here is that the assembly of every single test changes, potentially, on every different nightly. 

A better solution could be to update this build bot to the latest stable and update the tests to that. But I haven't tried if the latest stable fixes things.